src/app/app.component.ts

```diff
@@ -1,10 +1,36 @@
 import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
+import { map } from 'rxjs/operators';
+import { UserAuthenticationService } from './services/user-authentication.service';
+import { User } from './models';
+import { Router } from '@angular/router';
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'IT-academy-project';
+    title = 'Summer Good Deeds';
+    currentUser: User;
+
+    isHandset$: Observable<boolean> = this.breakpointObserver
+        .observe(Breakpoints.Handset)
+        .pipe(map(result => result.matches));
+
+    logout() {
+        this.userAuthenticationService.logout();
+        this.router.navigate(['/login']);
+    }
+
+    user() {
+        return localStorage.getItem('currentUser');
+    }
+
+    constructor(
```

Comment:
constructor should be higher up the component, definitely not at the bottom

Reply:
+